### ISSUE TITLE
Support For Way Generators 2.*

### ISSUE THREAD

**Issue by grafikkaos** [barryvdh/laravel-migration-generator#12]
Trying to add this package throws up an error with Jeffrey Way's generators as it has been upgraded to the 2.\* branch.  Will this package follow suite as well?

Thanks

**Comment by barryvdh** [comment]
What error? It should just require the 1.x release and still work, or are you trying to use the generator also stand-alone?
I will try to make this work on 2.0, but I don't really have a lot of time now.

**Comment by grafikkaos** [comment]
Hi,

The error I am getting is:

Please provide a version constraint for the barryvdh/laravel-migration-generator requirement: dev-master
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Installation request for barryvdh/laravel-migration-generator dev-master -> satisfiable by barryvdh/laravel-migration-generator[dev-master].
    - barryvdh/laravel-migration-generator dev-master requires way/generators 1.x -> no matching package found.

Potential causes:
- A typo in the package name
- The package is not available in a stable-enough version according to your minimum-stability setting
  see https://groups.google.com/d/topic/composer-dev/_g3ASeIFlrc/discussion for more details.

**Comment by barryvdh** [comment]
But do you also require way/generator in your own composer?
Op 7 mrt. 2014 16:36 schreef "Emmanuel Okorie" notifications@github.com:

**Comment by grafikkaos** [comment]
Yeah I have, i have required "way/generators" : "2.*".  I suppose my only way round this would be to require version 1.x for the time being.

**Comment by barryvdh** [comment]
Yes I would just generate the migrations from db with the old version and
then just switch to 2.X
Or do you need this package after the initial import?
Op 7 mrt. 2014 17:01 schreef "Emmanuel Okorie" notifications@github.com:

**Comment by grafikkaos** [comment]
I can make do with version 1.x.  I just want to generate the migrations so if switching back to the old version is what I need to for the time bing, I am quite happy to do that.  

Many thanks for your replies.

**Comment by anheru88** [comment]
Hi barryvdh please can you change or update your composer.json.

you onli need rewrite  "way/generators": "1.x" for  "way/generators": ">=1.x", I think so, maybe this solve the problem.

**Comment by barryvdh** [comment]
No this package isn't compatible with 2.X yet.

**Comment by Omranic** [comment]
@barryvdh Is there any ETA when it will be compatible with 2.x?

**Comment by barryvdh** [comment]
Not really. But feel free to submit a PR.
In the meantime, I think you can just require this package, import your database and remove this package and use the new generators for the rest. Or is anyone using this after initial import?
I do want to upgrade to 2.x, because it probably fixes some problems with missing types or newly added functions, but I don't really have much spare time..

**Comment by barryvdh** [comment]
I just came across this package: https://github.com/Xethron/migrations-generator
Maybe this can help you?

**Comment by crypto-sporidium** [comment]
This might sound like an odd question but how do I remove way/generators ?

If I remove the dependency from my composer.json and run a composer update it removes the package just fine... 

The problem is they appear to have modified artisan and are now required so I cant run artisan after removing them ? 

Probably a better question is how do I get a fresh install of artisan so I can remove way/generators ?

**Comment by barryvdh** [comment]
You can remove the service provider from Your app config. You can also
change the contents of the artisan file with a fresh one from
laravel/laravel on github.
Op 21 mei 2014 03:42 schreef "abelgo" notifications@github.com:

**Comment by msauditech** [comment]
What I did is to go manually add this to the composer.json file
"require-dev": {
    "way/generators": "2.*"
    }

then run composer update --dev

**Comment by michaeljhopkins** [comment]
I temporarily changed what I had in my composer.json `"way/generators": "2.*",` to `"way/generators": "1.*",`. However +1 for 2.\* support
